crates/editor/src/editor.rs

```diff
@@ -16380,21 +16397,31 @@ impl Editor {
                     })
                     .context("buffer title")?;
 
-                let opened = workspace
-                    .update_in(acx, |workspace, window, cx| {
-                        Self::open_locations_in_multibuffer(
-                            workspace,
-                            locations,
-                            title,
-                            split,
-                            MultibufferSelectionMode::First,
-                            window,
-                            cx,
-                        )
-                    })
-                    .is_ok();
-
-                anyhow::Ok(Navigated::from_bool(opened))
+                workspace.update_in(acx, |workspace, window, cx| {
+                    let tag = locations.iter().find_map(|loc_link| {
+                        loc_link.origin.as_ref().map(|origin| {
+                            origin.buffer.read_with(cx, |buffer, _cx| {
+                                let snapshot = buffer.snapshot();
+                                snapshot
+                                    .text_for_range(origin.range.clone())
+                                    .collect::<String>()
+                            })
+                        })
+                    });
+                    let Some(ed) = Self::open_locations_in_multibuffer(
+                        workspace,
+                        locations.iter().map(|item| item.target.clone()).collect(),
+                        title,
+                        split,
+                        MultibufferSelectionMode::First,
+                        window,
+                        cx,
+                    ) else {
+                        return Navigated::No;
+                    };
+                    ed.read(cx).create_tag_stack_entry(tag, cx);
```

Comment:
We chatted on Tuesday about what to do in the case where go-to-definition is ambiguous, and Zed opens a multibuffer of all of the options. I decided that a Good Enough™ solution was to treat the multibuffer itself as the jump target. We can improve this if we'd like, but honestly it feels fine to me